cocos/render-scene/scene/camera.ts

```diff
@@ -31,6 +31,7 @@ import { RenderWindow } from '../core/render-window';
 import { GeometryRenderer } from '../../rendering/geometry-renderer';
 import { PostProcess } from '../../rendering/post-process/components/post-process';
 import type { Frustum } from '../../core/geometry';
+import { PipelineSettings } from '../../rendering/custom/settings';
```

Comment:
This `rendering/custom/settings` will not be feature-culled like custom pipeline.
It is exported by `rendering/index.ts` and `rendering/index.jsb.ts`.